frontend/src/pages/collab.js

```diff
@@ -277,8 +286,9 @@ function Collab() {
               </div>
               {isPartner
               ?
-              <div className="collab-chat-content">
-                u r matched with {matchedUser}
+              <div className="collab-chat-content" >
```

Comment:
also mb instead of removing the entire chat component and their chat history, can js show in the chat when the matched user connects / disconnects ?

Reply:
any issues on your end if i just remove the if else?

Reply:
should be fine its just ui but i feel like should show that the username disconnected somewhere